pkg/kn/commands/source/ping/update.go

```diff
@@ -93,7 +94,7 @@ func NewPingUpdateCommand(p *commands.KnParams) *cobra.Command {
 				updatedSource := b.Build()
 				return updatedSource, nil
 			}
-			backoff := retry.DefaultRetry
+			backoff := config.DefaultRetry
 			backoff.Steps = MaxUpdateRetries
```

Comment:
Do we need to actually set the `.Steps` on top of default value, since `MaxUpdateRetries` is changed to `5` everywhere.